app/controllers/sites/new.js

```diff
@@ -126,18 +123,21 @@ export default class CreateSitesNewController extends Controller {
     if (Object.keys(validationResult.errors).length > 0) {
       // Validation failed. Return
       this.validationErrors = validationResult.errors;
+      this.saveButtonPressed = 0;
+      this.hasError = true;
       return;
     }
-
     if (Object.keys(validationResult.warnings).length > 0) {
       this.saveButtonPressed = this.saveButtonPressed + 1;
+      this.hasError = false;
+      this.hasWarning = true;
+      console.log('Dit is je warning', this.hasWarning);
       if (this.saveButtonPressed === 2) {
```

Comment:
Zet saveButtonPressed terug op 0. Controllers blijven leven in je ember app. Dus 't kan zijn dat je bij de volgende keer dat je op de page komt je value 2 is en niet 0